internal/backend/db/dbgorm/projects.go

```diff
@@ -28,6 +30,44 @@ func (db *gormdb) CreateProject(ctx context.Context, p sdktypes.Project) error {
 	return translateError(db.createProject(ctx, project))
 }
 
+func (db *gormdb) deleteProject(ctx context.Context, projectID string) error {
+	p := scheme.Project{ProjectID: projectID}
+	return db.db.WithContext(ctx).Delete(&p).Error
+}
+
+func (db *gormdb) deleteProjectAndDependents(ctx context.Context, projectID string) error {
+	return db.transaction(ctx, func(tx *tx) error {
+		depsEnvs, err := db.getProjectDeployments(ctx, projectID)
+		if err != nil {
+			return err
+		}
+		var deplIDs, envIDs []string
+		for _, de := range depsEnvs {
+			if de.State != int32(sdktypes.DeploymentStateInactive.ToProto()) {
```

Comment:
does the new sdktypes provide a better way to do it?
like parse deployment state or something? I'm not sure
I tend to dislike this part:
```
 int32(sdktypes.DeploymentStateInactive.ToProto())
```

Reply:
I'm not sure myself. What's the new way to get int32 from sdktype enum?